isis/src/base/apps/handmos/handmos.xml

```diff
@@ -436,6 +467,11 @@
       group in the cube labels which could impact processing scripts/pipelines. Fixes #1620.
       References #1623.
     </change>
+    <change name="Summer Stapleton" date="2018-08-13">
+      Updated code and documentation to reflect new handling of tracking capabilities with an 
+      external tracking cube as well as clarify why special pixel flags are required when 
+      priority=ontop for multiband mosaics. References #2092
+    </change>

```

Comment:
I really like how clear the special pixel documentation is

Reply:
Thank you. :-)